python/S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT/S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT.PY

```diff
@@ -0,0 +1,390 @@
+# This file made available under CC0 1.0 Universal (https://creativecommons.org/publicdomain/zero/1.0/legalcode)
+#
+# Created with the Rule Development Kit: https://github.com/awslabs/aws-config-rdk
+#
+
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  S3_PUBLIC_ACCESS_SETTINGS_FOR_ACCOUNT
+Description:
+  Checks whether AWS S3 public access account settings match the assigned parameters.
+Trigger:
+  Periodic : 1 hour
+Resource Type to report on:
+  AWS::::Account
+Rule Parameters:
+  | ---------------------- | ---------- | ----------------------------------------------------------------------------------------|
+  | Parameter Name         | Type       | Description                                                                             |
+  | ---------------------- | ---------- | ----------------------------------------------------------------------------------------|
+  | BlockPublicAcls        | Mandatory  | Block new public ACLs and uploading public objects (True/False)                         |
+  | IgnorePublicAcls       | Mandatory  | Remove public access granted through public ACLs (True/False)                           |
+  | BlockPublicPolicy      | Mandatory  | Block new public bucket policies (True/False)                                           |
+  | RestrictPublicBuckets  | Mandatory  | Block public and cross-account access to buckets that have public policies (True/False) |
+  | ---------------------- | ---------- | ----------------------------------------------------------------------------------------|
+Feature:
+  In order to: ensure that s3 account settings are being restricted to the appropriate level
+           As: a Security Officer
+       I want: to verify that the configuration of s3 account public access settings is correct.
+Scenarios:
+  Scenario 1:
+    Given: config parameters are defined for the config rule
+      And: At least 1 S3 config parameter does not match the corresponding value for the account
+     then: Return NON_COMPLIANT
+  Scenario 2:
+    Given: config parameters are defined for the config rule
+      And: All S3 config parameters match the corresponding value for the account
+     then: Return COMPLIANT
+'''
+
+import json
+import datetime
+import os
+import os.path
+import sys
+envLambdaTaskRoot = os.environ["LAMBDA_TASK_ROOT"]  
+sys.path.insert(0,envLambdaTaskRoot+"/boto3-1-9-82") #hack to change the version of boto
+import boto3
+import botocore
+
+##############
+# Parameters #
+##############
+
+# Define the default resource to report to Config Rules
+DEFAULT_RESOURCE_TYPE = 'AWS::::Account'
+
+# Set to True to get the lambda to assume the Role attached on the Config Service (useful for cross-account).
+ASSUME_ROLE_MODE = False
+
+#############
+# Main Code #
+#############
+
+def evaluate_compliance(event, configuration_item, valid_rule_parameters):
+  
+    client = boto3.client('s3control')

```

Comment:
use the helper function `get_client()` to be compatible with multi-account deployment

Reply:
replaced boto3.client call with:

client = get_client('s3control', event)